tests/components/drop_connect/test_sensor.py

```diff
@@ -64,49 +70,47 @@ async def test_sensors_hub(
 
     current_flow_sensor = hass.states.get(current_flow_sensor_name)
     assert current_flow_sensor
-    assert round(float(current_flow_sensor.state), 1) == 5.8
+    assert current_flow_sensor.state == "5.77"
```

Comment:
The use of round() was an attempt to avoid subtle precision errors on varying platforms from unnecessarily failing the tests.  If that is not an actual issue, I'm fine with removing them.  The expectation that floating point math could be inconsistent across platforms is a bias I've picked up from the embedded microcontroller realm and may not be applicable here.

Reply:
We are testing against 2 fixed versions of python. And those should give a consistent result, even over multiple platforms.

Reply:
Worst case scenario there is also a function to compare a value with a variance